tests/parsing/ConfParserTest.py

```diff
@@ -200,3 +200,29 @@ def test_settings_override_warning(self):
                                             'already been defined in section '
                                             'name. The previous setting will '
                                             'be overridden.')
+
+
+class ConfParserTestParseSections(unittest.TestCase):
+    test_file = """
+    [Default]
+    ignore = aaa,bbb
+    ignore += ccc
+    """
+
+    def setUp(self):
+        self.tempdir = tempfile.gettempdir()
+        self.file = os.path.join(self.tempdir, '.coafile')
+        with open(self.file, 'w') as file:
+            file.write(self.test_file)
+        self.uut = ConfParser()
+        self.sections = self.uut.parse(self.file)
+
+    def tearDown(self):
+        os.remove(self.file)
+
+    def test_multiple_ignore_in_single_settings(self):
+        expected = ['aaa', 'bbb', 'ccc']
+        self.check = self.uut.sections['default'].contents['ignore'].value.\
```

Comment:
Same here

Reply:
dont save things into `self.` inside a method, unless it is going to be used outside of that method.

in unit tests, typically `self.` is only populated inside `setUp` and `tearDown`.